### ISSUE TITLE
Depending on github for vector_math causes conflicts

### ISSUE THREAD

**Issue by jimbeveridge** [fkleon/math-expressions#4]
In revision 961ba77, the following change was made to pubspec.yaml:
```
-  vector_math: '>=1.3.5 <1.5.0'
+  vector_math:
+    git: git://github.com/google/vector_math.dart.git
```

The new dependency on github causes projects that depend on math-expressions to use the older version 0.1.0 if there are other packages that depend on the "normal" version of vector_math. Although the problem can be fixed with override_dependencies, the problem is subtle and painful to debug.

**Comment by jimbeveridge** [comment]
@johnmccutchan

**Comment by fkleon** [comment]
Thanks for this report - you are absolutely right. I had put a warning regarding this in the [changelog](https://github.com/fkleon/math-expressions/blob/89ec90b725d22f8d4deb6689b02dcb1d44ea2c2c/CHANGELOG.md) for `0.2.0`, but this was clearly not an optimal change.

Some of the recent changes for `math-expressions` depend on features added since the latest pub release of [vector_math](https://github.com/google/vector_math.dart) - which is actually quite outdated. I've created the issue google/vector_math.dart#157 to track this.

**Comment by johnmccutchan** [comment]
vector_math 1.4.4 has now been published.

**Comment by fkleon** [comment]
Great, thanks @johnmccutchan! Will update `math-expressions` tonight to depend on that new version instead.

**Comment by fkleon** [comment]
I have released math_expressions 0.2.0+1 to pub.
